src/components/Modal/Modal.vue

```diff
@@ -674,7 +674,7 @@ $header-size: 50px;
 
 	/* Content */
 	.modal-container {
-		display: block;
```

Comment:
No, please keep it as default html: block.
This is a breaking change.
If you want it flex, change it on your app.

Unless there is something I missed, but I don't think enforcing flex is a good idea. We should still stay close as the html basics I think :)

Reply:
I made it configurable, defaults to false.

This is needed for any instance of AppSettingsDialog for smaller screen resolutions.

Reply:
Then override it with v-deep.
That is why it's there for :see_no_evil: